index.js

```diff
@@ -16,6 +16,9 @@ module.exports = {
         core.helper.simpleWrapper(
             core.helper.requireGlob(__dirname + '/lib/task-data/**/*.js'),
             'Task.taskLibrary'
-        )
+        ),
+         core.helper.requireWrapper('superagent', 'Superagent')
```

Comment:
The `requireWrapper` always need to explicitly pass the directory name if this is not `on-core` repo, we had found issues due to this multiple times before. 
This is the correct usage:  https://github.com/RackHD/on-tasks/blob/master/index.js#L11